### PR TITLE
Changes to Oozie site xml to enable email action

### PR DESCRIPTION
**Pull request by bijugs**
This is for enhancement req in #679.

**Note:** Not tested by creating a new cluster.

### CODE REVIEW

File: cookbooks/bcpc-hadoop/templates/default/ooz_oozie-site.xml.erb [hunk 1]
```diff
@@ -384,4 +384,21 @@
...
 
+  <% if node['bcpc']['hadoop']['oozie']['smtp_host'] != nil then %> 
+
+  <property>
+    <name>oozie.email.smtp.host</name>
+    <value><%= node['bcpc']['hadoop']['oozie']['smtp_host'] %></value>
+  </property>
+
+  <property>
+    <name>oozie.email.from.address</name>
```
Comment (cbaenziger) [inline cookbooks/bcpc-hadoop/templates/default/ooz_oozie-site.xml.erb]: Is there anyway we can set this to something akin to `${wf:user()}`?

Comment (bijugs) [inline cookbooks/bcpc-hadoop/templates/default/ooz_oozie-site.xml.erb]: AFAIK... currently the from e-mail id can be set to only a constant value.

**Comment by bijugs** [comment]
anything else required here?

**Comment by bijugs** [comment]
This will be a good feature to have for users. Anything else needs to be done here?

**Comment by bijugs** [comment]
Thanks @vt0r .

**Pull request merged**